src/pages/steps/OrganizerStep.tsx

```diff
@@ -21,12 +38,36 @@ const OrganizerStep = ({
   onSuccessfulChange,
   ...props
 }: Props) => {
-  const { i18n } = useTranslation();
+  const { t, i18n } = useTranslation();
+  const queryClient = useQueryClient();
 
   const getEventByIdQuery = useGetEventByIdQuery({ id: eventId });
 
+  // @ts-ignore
+  const getCardSystemForEventQuery = useGetCardSystemForEventQuery({
+    eventId,
+  });
+
+  // @ts-expect-error
+  const event: Event | undefined = getEventByIdQuery.data;
+
+  const organizer = event?.organizer;
+  const hasPriceInfo = (event?.priceInfo ?? []).length > 0;
+
+  // @ts-ignore
```

Comment:
```suggestion
// @ts-expect-error
```

use @ts-expect-error so typescript will tell you when the comment is no longer needed